tests/src/test/scala/org/http4s/parser/ContentDispositionHeaderSuite.scala

```diff
@@ -41,4 +44,29 @@ class ContentDispositionHeaderSuite extends Http4sSuite {
     )
   }
 
+  property("ContentDisposition filename encoding roundtrip") {
```

Comment:
It'd be good to add a few particular test cases when the previous parser fails and the new one doesn't.

Reply:
Done